src/kvstore/raftex/test/LogAppendTest.cpp

```diff
@@ -29,32 +29,17 @@ TEST(LogAppend, SimpleAppendWithOneCopy) {
     std::vector<HostAddr> allHosts;
     std::vector<std::shared_ptr<RaftexService>> services;
     std::vector<std::shared_ptr<test::TestShard>> copies;
+    std::vector<LogID> lastCommittedLogId;
```

Comment:
What's the purpose?

Reply:
It is used to store a copy's last commit log id outside the Part itself (just like the kLastCommittedIdKey we store it in rocksdb). When a TestShard reboot we can get lastCommittedLogId here.

Reply:
If so, why not put it in TestShard?

Reply:
Good suggestion